submission/uDSR/environment.yml

```diff
@@ -0,0 +1,90 @@
+name: udsr
```

Comment:
I suggest removing as much of the package versioning as you can to help conda find a matching set of dependencies.

Reply:
Will try that, thanks. Working on paring down a minimal dependency version. Some of the conflicts seem odd though, e.g. `package argon2-cffi-bindings-21.2.0-py39hb9d737c_2 requires python >=3.9,<3.10.0a0`, we don't have `argon2` on our dependency list -- is it on the SRBench dependency list? Is the base environment trying to combine envs from all the other submissions?

Reply:
I think you might need to use
```yml
channels:
    - conda-forge
```
Removing versions should help a lot.

Reply:
the environments of each submission are separate, but your code has to play well with the srbench dependencies [here](https://github.com/cavalab/srbench/blob/Competition2022/environment.yml)

not sure of `argon2-cffi`

Reply:
@lacava Those linked dependencies work fine.

@foolnotion Thanks, I will add `conda-forge` and remove lots of versions. I don't know how to test this though other than just updating the PR and crossing my fingers...

Reply:
@lacava It just failed with a dirt-simple environment file consisting of:

name: udsr
channels:
  - defaults
  - conda-forge
dependencies:
  - python=3.6
```

Seems like there is some other stuff in the base `srcomp`?

Reply:
Could you try removing the `defaults` channel or putting it after `conda-forge`? Packages from different channels don't mix well.

Reply:
@foolnotion Done. Same error. Is this issue possibly unrelated to my submission?

Reply:
@lacava @foolnotion I mucked around with the versions of things until it worked. I guess the base `srcomp` really does not like `python3.6` for some reason. But it's working for me now with `python3.7`. Thanks again for your help and prompt replies :)